plugins/glustershd/rest.go

```diff
@@ -0,0 +1,167 @@
+package glustershd
+
+import (
+	"net/http"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/errors"
+
+	"github.com/gorilla/mux"
+	"github.com/pborman/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+func isVolReplicate(vType volume.VolType) bool {
+	if vType != volume.Replicate && vType != volume.Disperse && vType != volume.DistReplicate && vType != volume.DistDisperse {
```

Comment:
You can inverse the first if condition and return true. And then return false for everything else. That makes it easier to understand.

```go
if vType == volume.Replicate || vType == volume.Disperse.... {
    return true
}

return false
```